__tests__/paddle.ts

```diff
@@ -935,180 +935,6 @@ describe('pricing preview by ids', () => {
   });
 });
 
-describe('core pricing preview', () => {
-  const QUERY = /* GraphQL */ `
-    query CorePricingPreview($type: PricingType, $locale: String) {
-      corePricingPreview(type: $type, locale: $locale) {
```

Comment:
There is really no query `corePricingPreview`. We use the same query for Plus then send a different type :hide_the_pain:

Of course we have an existing test case for that.